Firmware/ChameleonMini/Makefile

```diff
@@ -62,17 +66,17 @@ ARCH		 = XMEGA
 BOARD		 = NONE
 F_CPU		 = 32000000
 F_USB		 = 48000000
-TARGET		 = ChameleonMini
+TARGET		 = Chameleon-RevE
```

Comment:
I don't think you should do this change.   See next change below, why.

Reply:
In my opinion,the TARGET flag means firmware hex filename and the ChameleonMini.c is the fw src main entry.It is not the same thing.we shouldnt change the main entry."Chameleon-RevE" might more clearly.I think it should be common and general
the OBJDIR flag has a default value "obj",the value "Bin" should be the folder compiled hex files I think.

Reply:
although,I just thought it is convenience.If it confused you im sorry and I can revert it back

Reply:
It would be nice to see a better naming convention for the artifact.   If so,  when it should reflected in all diff wiki pages /documentations :)

Which would mean another PR...

Reply:
yeah thats another PR.so I just revert the TARGET flag right now,and maybe create another PR to make things better :)

Reply:
you would need to revert the line you changed below,   which used the TARGET ;)

Reply:
sure.my bad ;(